examples/constraints/coneTwist.html

```diff
@@ -0,0 +1,26 @@
+
+<html>
+  <head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no,user-scalable=no,maximum-scale=1">
+    <title>Distance Constraint Example</title>
+    <script type="text/javascript" src="https://aframe.io/releases/0.5.0/aframe.min.js"></script>
+    <script type="text/javascript" src="../../dist/aframe-physics-system.js"></script>
```

Comment:
All examples should point to: `../bundle.js`. With that change, you can just run `npm run dev` to start a local server that updates automatically.